src/executor.rs

```diff
@@ -58,7 +60,9 @@ impl From<SubMsgResponse> for AppResponse {
         }
     }
 }
-
+///Defines the interface for executing transactions and contract interactions. 
```

Comment:
Whitespace needed between `///` an the text. Trailing space at the end of the line should be removed.